website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -2893,6 +2893,60 @@ No response body.
 
 Status code `204`.
 
+## Show Data Retention Policy
+
+<EnterpriseAlert>
+This endpoint is exclusive to Terraform Enterprise, and not available in Terraform Cloud.
+</EnterpriseAlert>
+
+`GET /workspaces/:workspace_id/relationships/data-retention-policy`
+
+| Parameter            | Description                                                         |
+| ---------------------| --------------------------------------------------------------------|
+| `:workspace_id`      | The ID of the workspace to show the data retention policy for. Obtain this from the [workspace settings](/terraform/cloud-docs/workspaces/settings) or the [Show Workspace](#show-workspace) endpoint. |
+
+
+This endpoint shows the data retention policy set explicitly on the workspace.
+When no data retention policy is set on the workspace, the organization data retention policy will be used by default. Read more about [workspace data retention policies](/terraform/cloud-docs/workspaces/settings/deletion#data-retention-policies).
```

Comment:
Non-blocking nitpick (same as above!): Should we add an extra line between this line of text and the previous line of text to keep it consistent with the [text on line 948](https://github.com/hashicorp/terraform-docs-common/pull/556/files#diff-f5625a4be7f4b34305dd570e3a8eb331c34e7fd7bc58dab8f5499ed46ceb8737R948)?:
```suggestion
When no data retention policy is set on the workspace, the organization data retention policy will be used by default. 

Read more about [workspace data retention policies](/terraform/cloud-docs/workspaces/settings/deletion#data-retention-policies).
```